src/events/sql.js

```diff
@@ -96,9 +96,13 @@ module.exports.wrapSqlQuery = function wrapSqlQuery(queryString, params, callbac
                 if (err) {
                     eventInterface.setException(dbapiEvent, err);
                 } else {
-                    let { rowCount } = res;
+                    let { rowCount, rows } = res;
```

Comment:
This would be nicer (and clearer) if calling `addToMetadata(... { 'sql.rows': ...});` would happen in a single place. 
Something like that: 
```
let { rowCount, rows } = res;
if (!rowCount && res instanceof Array) {
    rowCount = res.length
    rows = res
}

eventInterface.addToMetadata(dbapiEvent, { rowCount });
if (rowCount && rows instanceof Array && rows.length) {
    eventInterface.addToMetadata(dbapiEvent, { 'sql.rows': rows });
}
```

Reply:
Done